docs/configs/janusgraph-cfg.md

```diff
@@ -437,7 +437,7 @@ CQL storage backend options
 | storage.cql.compaction-strategy-options | Compaction strategy options.  This list is interpreted as a map.  It must have an even number of elements in [key,val,key,val,...] form. | String[] | (no default value) | FIXED |
 | storage.cql.compression | Whether the storage backend should use compression when storing the data | Boolean | true | FIXED |
 | storage.cql.compression-block-size | The size of the compression blocks in kilobytes | Integer | 64 | FIXED |
-| storage.cql.compression-type | The sstable_compression value JanusGraph uses when creating column families. This accepts any value allowed by Cassandra's sstable_compression option. Leave this unset to disable sstable_compression on JanusGraph-created CFs. | String | LZ4Compressor | MASKABLE |
+| storage.cql.compression-type | The compression class value JanusGraph uses when creating column families. This accepts any value allowed by Cassandra's compression class option. Leave this unset to disable compression class on JanusGraph-created CFs. | String | LZ4Compressor | MASKABLE |
```

Comment:
build-doc CI fails because in the last sentence here you specified `compression class` but in configuration option only `compression` is specified. You can execute `mvn --quiet clean install -DskipTests=true -pl janusgraph-doc -am` to automatically re-generate this file, so that `compression class` automatically turns into `compression`.